src/app.ts

```diff
@@ -1,60 +1,70 @@
 import fastify, { FastifyPluginAsync } from 'fastify';
 import fp from 'fastify-plugin';
 import {
-	PG_CONNECTION_URI, DATABASE_LOGS, DISABLE_LOGS,
-	ENVIRONMENT, MAILER_CONFIG_SMTP_HOST, MAILER_CONFIG_USERNAME, MAILER_CONFIG_PASSWORD, MAILER_CONFIG_FROM_EMAIL
+  PG_CONNECTION_URI,

```

Comment:
@pyphilia, are all these changes b/c @HadiKhai's formatting is different?

Reply:
I think my formatting is different from André's, which I guess match Hadi's.
This PR is mainly here in case you wanted to deploy superuser when I couldn't.